include/SDL3/SDL_events.h

```diff
@@ -1400,6 +1400,20 @@ extern SDL_DECLSPEC SDL_bool SDLCALL SDL_EventEnabled(Uint32 type);
  */
 extern SDL_DECLSPEC Uint32 SDLCALL SDL_RegisterEvents(int numevents);
 
+/**
+ * Get the associated window id of an event.
+ *
+ * \param event SDL event of which to get the window id.
+ *
+ * \returns the associated window id on success or 0 if there is no associated window id.
+ *
+ * \since This enum is available since SDL 3.0.0.
```

Comment:
```suggestion
 * \since This function is available since SDL 3.0.0.
```

Reply:
Updated the documentation for `SDL_GetWindowFromEvent`.